.github/workflows/bb_containers.yml

```diff
@@ -76,37 +62,43 @@ jobs:
             image: ubuntu:20.04
             branch: 10.7
             platforms: linux/amd64, linux/arm64/v8, linux/ppc64le, linux/s390x
-          - dockerfile: debian.Dockerfile
-            image: ubuntu:21.10
-            branch: 10.7
-            platforms: linux/amd64, linux/arm64/v8, linux/ppc64le, linux/s390x
           - dockerfile: debian.Dockerfile
             image: ubuntu:22.04
             branch: 10.7
             platforms: linux/amd64, linux/arm64/v8, linux/ppc64le, linux/s390x
-          - dockerfile: fedora.Dockerfile
-            image: fedora:34
-            platforms: linux/amd64, linux/arm64/v8
           - dockerfile: fedora.Dockerfile
             image: fedora:35
             platforms: linux/amd64, linux/arm64/v8
           - dockerfile: fedora.Dockerfile
             image: fedora:36
             platforms: linux/amd64, linux/arm64/v8
+# conflicts - see https://github.com/MariaDB/buildbot/runs/7106820431?check_suite_focus=true#step:11:2205
+#          - dockerfile: fedora.Dockerfile
+#            image: fedora:37
+#            platforms: linux/amd64, linux/arm64/v8
```

Comment:
Not if buildbot-worker is provided by the distribution, names are as follow:
- python3-buildbot-worker (debian based);
- buildbot-worker (rhel based).

pip is not reliable so better to avoid it if possible.

Reply:
I assume that the packages were not updated accordingly for fedora:37. Or is there any other explanation for the failure from https://github.com/MariaDB/buildbot/runs/7106820431?check_suite_focus=true#step:11:2205?

Reply:
I didn't look into it. Left it for another date closer to 37 release.